test/parallel/test-tls-key-mismatch.js

```diff
@@ -29,6 +29,9 @@ if (!common.hasCrypto) {
 const assert = require('assert');
 const tls = require('tls');
 const fs = require('fs');
+const errorMessageRegex = new RegExp('^Error: error:0B080074:x509 ' +
```

Comment:
Is it possible write this in a way that involves less line-breaking? Its all unused whitespace ATM :-(. Maybe put the msg string as its own line, `const msg = 'error:0B....';`? Or wrapping the entire argument to `RegExp()` to the next line?